SetupTearDown.ps1

```diff
@@ -29,6 +41,122 @@ Function SetupRemoting{
 
 } # SetupRemoting()
 
+Function SetupSshRemotingOnHost{

```

Comment:
any reason we are hard-coding the .pub?  more extensible if we were to take in a parameter

Reply:
I was following the documentation [here](https://learn.microsoft.com/en-us/windows-server/administration/openssh/openssh_keymanagement#user-key-generation), but I can change it so the user passes in the key paths.